tests/testthat/test-derive_vars_dtm.R

```diff
@@ -792,3 +792,5 @@ test_that("derive_vars_dtm Test 31: catch ignore_seconds_flag error", {
     error = TRUE
   )
 })
+
+rm(list = c("input"))
```

Comment:
ha so easy.

Maybe we should adopt this test data approach for all our tests. https://r-pkgs.org/testing-design.html#storing-test-data  It would really cut down on them being long files.  probably a larger discussion.  

then we wouldn't need the rm() as everybody has there own test data to pull from.